### DISCUSSION TITLE
How to edit a skin so that lazer fallbacks to lazer gameplay elements instead of osu!classic when a texture is missing from the skin?

### DISCUSSION THREAD

**Discussion by ArchUsr64** [ppy/osu#13754] [Q&A] [answered]
I use a skin that relies on default textures for the most part and just changes a few gameplay elements and was made last year, it worked well for each lazer update till May.
This [change](https://github.com/ppy/osu/issues/13448) seems to be the culprit and to my understanding, it looks like the game is relying on osu!classic textures for all the missing elements from the skin, is there any way I change this behavior?

Additional media:
How the skin is supposed to look(screenshot from 2021.424.0)
![DeepinScreenshot_osu!_20210703132223](https://user-images.githubusercontent.com/83179501/124348256-86896d80-dc06-11eb-9884-e9dc6d74f593.png)
How it looks ever since 2021.424.0 including the latest 2021.703.0
![DeepinScreenshot_osu!_20210703131924](https://user-images.githubusercontent.com/83179501/124348316-eda72200-dc06-11eb-9598-302b853dce12.png)
The skin in question(changed extension to zip to upload on Github)
[ClutterFree (Me).zip](https://github.com/ppy/osu/files/6758088/ClutterFree.Me.zip)

**Comment by peppy** [comment, marked as answer]
This will eventually be better supported, but is currently out of scope. You can use the new built-in skin editor to customise the default skin, but changing textures is not yet supported.
